### ISSUE TITLE
creating dup collections

### ISSUE THREAD

**Issue by paulserraino** [goodybag/loglog-mongodb#2]
loglog is attempting to create a second 'log' collection
![screen shot 2015-01-26 at 1 57 03 pm](https://cloud.githubusercontent.com/assets/3300063/5907293/6de77a46-a563-11e4-883d-118398f8ff3d.png)

We could opt to just using `insert()`. If the collection does not exist, then the insert() method will create the collection.

http://docs.mongodb.org/manual/reference/method/db.collection.insert/

**Comment by jrf0110** [comment]
ah, good call

**Comment by jrf0110** [comment]
well, actually - we can't do that

**Comment by jrf0110** [comment]
We need to be able to pass options to the collection (through createCollection)

**Comment by jrf0110** [comment]
If the user needs a capped collection (commonly true for logging), then you need to call createCollection with options

**Comment by paulserraino** [comment]
ah, I see, maybe we could use something like `db.getCollection(options.collection).exists()`.

**Comment by jrf0110** [comment]
yeah, definitely need to do something like that. Can you submit a PR?

**Comment by paulserraino** [comment]
looks like `getCollection` only works in the mongo shell. I'll have to try something else :/
